packages/design-tokens/bin/buildCssVarTokens.ts

```diff
@@ -29,8 +30,18 @@ const run = async (): Promise<void> => {
   fs.mkdirSync(jsonOutput, { recursive: true })
   fs.mkdirSync(cssOutput, { recursive: true })
 
+  // Write CSS vars tokens to CSS format
+  const cssVars = toCustomPropertiesString(
+    makeCssVariableDefinitionsMap(defaultTheme)
+  )
+
+  fs.writeFileSync(
+    path.resolve(cssOutput, "variables.css"),
+    `html {${cssVars}}`
+  )
+
   /**
-   *
+   * augmentedThemeWithCSSVariableValuesVersion
```

Comment:
Oh? Is this referencing a function?

Reply:
No idea haha, left it as is.

Reply:
Really? Cos it looks like you added it in 😂

Reply:
Oh wait, yeah it is below